packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -159,7 +161,7 @@ export interface DropdownProps extends UIComponentProps<DropdownProps, DropdownS
   multiple?: boolean;
 
   /** A message to be displayed in the list when the dropdown has no items. */
-  noResultsMessage?: ShorthandValue<ListItemProps>;
```

Comment:
Wouldn't that be a breaking change for consumers since ListItemProps and DropdownItemProps have different interfaces?